images/metering-ansible-operator/roles/meteringconfig/tasks/configure_reporting_operator_tls.yml

```diff
@@ -117,8 +117,11 @@
 
   - name: Generate cookie seed and configuring authProxy to use that generated seed value
     block:
+      # Note: need to use the 'shell' module here as the 'command' module does
+      # not support using pipes, and I wasn't able to find a decent Ansible filter
+      # that truncated any characters > 32.
     - name: Generate a 32-character random string
-      command: openssl rand -base64 32
+      shell: openssl rand -base64 32 | head -c 32
```

Comment:
I think this is a result of the encoding chosen:

```metering-operator make_ci_like_rhel_docker_build $ openssl rand -base64 32|head -c 32|wc
       0       1      32
metering-operator make_ci_like_rhel_docker_build $ openssl rand -base64 32
J5RfxDqtb018N8QiJaXGEdZhPxJgSTsvuRWxgswe1NE=
metering-operator make_ci_like_rhel_docker_build $ openssl rand -base64 32|wc
       1       1      45
metering-operator make_ci_like_rhel_docker_build $ openssl rand -hex 32|wc
       1       1      65```
Truncating post-encoding might have bad implications for cryptographic soundness (I'm assuming we use that see for such purposes).  Can we just ditch the 32 char limit on the base64 encoded string?